values_test.go

```diff
@@ -13,7 +13,7 @@ func TestGetValues_error(t *testing.T) {
 	repo := newRepo(t)
 	defer os.RemoveAll(repo.dir)
 
-	_, err := GetFlags(repo.dir)
+	_, err := GetFlags(repo.dir, defaultPackage)
```

Comment:
please pass these as `, "main"` explicitly because we're validating it against `"main.Xxx"` strings below.